spec/models/employment_appeal_tribunal_decision_spec.rb

```diff
@@ -0,0 +1,181 @@
+      "format" => "placeholder_organisation",
+      "need_ids" => [],
+      "locale" => "en",
+      "updated_at" => "2015-08-20T10:26:56.082Z",
+      "public_updated_at" => "2015-04-15T10:04:28.000+00:00",
+      "phase" => "live",
+      "analytics_identifier" => "CO1134",
+      "links" => {
+        "available_translations" => [
+          {
+            "content_id" => "975cf540-6e64-40e3-b62a-df655a8c99ef",
+            "title" => "Employment appeal tribunal decisions",
+            "base_path" => "/employment-appeal-tribunal-decisions",
+            "description" => nil,
+            "api_url" => "https://www-origin.integration.publishing.service.gov.uk/api/content/employment-appeal-tribunal-decisions",
+            "web_url" => "https://www-origin.integration.publishing.service.gov.uk/employment-appeal-tribunal-decisions",
+            "locale" => "en"
+          }
+        ]
+      },
+      "description" => nil,
+      "details" => {}
+    }
+  }
+
+  let(:indexable_attributes) {
+    {
+      "title" => "Example Employment Appeal Tribunal Decision 0",
+      "description" => "This is the summary of example Employment Appeal Tribunal Decision 0",
+      "link" => "/employment-appeal-tribunal-decisions/example-employment-appeal-tribunal-decision-0",
+      "indexable_content" => "## Header" + ("\r\n\r\nThis is the long body of an example Employment Appeal Tribunal Decision" * 10),
+      "public_timestamp" => "2015-11-16T11:53:30+00:00",
+      "tribunal_decision_categories" => ["age-discrimination"],
+      "tribunal_decision_decision_date" => "2015-07-30",
+      "tribunal_decision_landmark" => "landmark",
+      "tribunal_decision_sub_categories" => ["contract-of-employment-apprenticeship"],
+      "organisations" => ["employment-appeal-tribunal"],
+    }
+  }
+
+  let(:fields) { %i[base_path content_id] }
+
+  let(:employment_appeal_tribunal_decisions) { 10.times.map { |n| employment_appeal_tribunal_decision_content_item(n) } }
+
+  before do
+    publishing_api_has_fields_for_format('specialist_document', employment_appeal_tribunal_decisions, fields)
+
+    employment_appeal_tribunal_decisions.each do |decision|
+      publishing_api_has_item(decision)
+    end
+
+    Timecop.freeze(Time.parse("2015-12-18 10:12:26 UTC"))
+  end
+
+  describe "#all" do
+    it "returns all Employment Appeal Tribunal Decisions" do
+      expect(described_class.all.length).to be(employment_appeal_tribunal_decisions.length)
+    end
+
+    it "rejects any non Employment Appeal Tribunal Decisions" do
+      all_specialist_documents = [non_employment_appeal_tribunal_decision_content_item] + employment_appeal_tribunal_decisions
+      publishing_api_has_fields_for_format('specialist_document', all_specialist_documents , fields)
+      publishing_api_has_item(non_employment_appeal_tribunal_decision_content_item)
+
+      expect(described_class.all.length).to be(employment_appeal_tribunal_decisions.length)
+    end
+  end
+
+  describe "#find" do
```

Comment:
Just realised this is referring to a class method. The standard Ruby notation is for class methods to use a dot, and instance methods to use a hash. So this should be `describe ".find"` and below should stay `describe "#save!"`. Sorry for not spotting this before.